pmd-java/src/main/resources/category/java/security.xml

```diff
@@ -60,6 +60,34 @@ public class Foo {
         byte[] iv = "secret iv in here".getBytes();
     }
 }
+]]>
+        </example>
+    </rule>
+
+    <rule name="StaticFieldMustBeFinal"
```

Comment:
Actually I wanted to make another rule that would subsume this... It's hinted at at the end of #2916.

That rule would be called `MutableStaticState`. It could just be doing the job of StaticFieldMustBeFinal in the beginning, but we could also generalize it to other constructs later, because the name is more generic.

So I would recommend that this rule be renamed to `MutableStaticState` so that we can extend it later.

Reply:
The rule is currently in the security category.... maybe "design" would be a better fit?